torch/csrc/jit/serialization/export.cpp

```diff
@@ -1366,11 +1366,8 @@ void check_onnx_proto(const std::string& proto_string, bool full_check) {
     throw std::runtime_error("Invalid ONNX proto string.");
     return;
   }
-  onnx::checker::check_model(model);
-
-  if (full_check) {
-    onnx::shape_inference::InferShapes(model);
```

Comment:
why was this removed?

Reply:
onnx::shape_inference::InferShapes(model) is included in onnx::checker::check_model(model) in the latest version.
https://github.com/onnx/onnx/blob/f0d824c0ead2825f7c24790c05f985ad1fb909f2/onnx/checker.cc#L941-L958

Reply:
I thought check_model includes regular InferShapes, but it's not. I will later change it to:
```python
onnx::checker::check_model(model, full_check);
if (!full_check) {
  onnx::shape_inference::InferShapes(model);
}
```

Reply:
Done